core/rpc/api/src/author/mod.rs

```diff
@@ -83,4 +83,24 @@ pub trait AuthorApi<Hash, BlockHash> {
 		metadata: Option<Self::Metadata>,
 		id: SubscriptionId
 	) -> Result<bool>;
+
+	/// Watch multiple extrinsics (own or from network)
+	#[pubsub(
+		subscription = "author_extrinsicTrack",
+		subscribe,
+		name = "author_trackExtrinsic"
+	)]
+	fn track_extrinsic(&self,
+    	metadata: Self::Metadata,
+		subscriber: Subscriber<Status<Hash, BlockHash>>,
+	    hash: Hash,
+	);
+
+	/// Unsubscribe from multiple extrinsics watching.
+	#[pubsub(
+		subscription = "author_extrinsicTrack",
+		unsubscribe,
+		name = "author_untrackExtrinsic"
```

Comment:
```suggestion
		name = "author_unwatchExtrinsic"
```

Have you tried naming the unsubscribe method the same? I guess that will hopefuly work, but in both methods we will need to unsubscribe from both subscriptions in such case.

Reply:
naming seems working, but how do you unsubscribe from both?

Reply:
I guess if we call the same `self.subscription.unsubscribe` it actually does unsubscribe from any of the two.